src/CLI/IOutputGenerator.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Axe.Windows.Automation;
+
+namespace AxeWindowsScanner
```

Comment:
The namespace is the name of the app. Happy to change things if there are more appropriate values